### PR TITLE
ETL assignment_II

### PR DESCRIPTION
**Pull request by sumansta**

### CODE REVIEW

File: day_2/day_2.egg-info/dependency_links.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (sumansta) [inline day_2/day_2.egg-info/dependency_links.txt:1]: Can you add these in .gitignore? @Nabinhyan

File: day_2/schema/create_raw_employee.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        employee_role VARCHAR(200),
+        salary VARCHAR(200),
+        hire_date VARCHAR(200),
+        terminated_date VARCHAR(200),
+        terminated_reason VARCHAR(200),
+        dob VARCHAR(200),
+        fte VARCHAR(200),
+        location VARCHAR(200)
+);
+
```
Comment (sumansta) [inline day_2/schema/create_raw_employee.sql:17]:
```suggestion
```
Remove unused lines.

File: day_2/src/sql/queries/_batch_raw_timestamp.sql [hunk 3]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO etl.raw_timestamp("employee_id","cost_center","punch_in_time","punch_out_time","punch_apply_date","hours_worked","paycode")
+VALUES
+(%s,%s,%s,%s,%s,%s,%s);
+INSERT 0 10
```
Comment (sumansta) [inline day_2/src/sql/queries/_batch_raw_timestamp.sql]: What is the use of this line? @Nabinhyan

File: day_2/src/sql/queries/delete_raw_timestamp.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+DELETE FROM etl.raw_timestamp;
```
Comment (sumansta) [inline day_2/src/sql/queries/delete_raw_timestamp.sql:1]: Add a new line at the end for every files. @Nabinhyan

File: day_2/day_2.egg-info/dependency_links.txt (hunk 1 above)
Comment (nabinhyan) [reply day_2/day_2.egg-info/dependency_links.txt:1]: ok dai 
i will do this on upcoming days
```suggestion

```

File: day_2/src/sql/queries/_batch_raw_timestamp.sql (hunk 3 above)
Comment (nabinhyan) [reply day_2/src/sql/queries/_batch_raw_timestamp.sql]: actually dai, 
that was my mistake not to delete the batch_raw_timestamp.sql fiel

**Pull request closed without merging**